client/lib/plans/constants.js

```diff
@@ -184,7 +184,7 @@ const getPlanPersonalDetails = () => ( {
 				},
 			}
 		),
-	getFeatures: () => [
+	getPlanListFeatures: () => [
```

Comment:
Renamed these to try and make it clearer that this is a subset of features for each plan, specifically used to populate the PlanList component.